### ISSUE TITLE
Performances

### ISSUE THREAD

**Issue by MattiSG** [agile-france/site-2015#5]
Améliorer la vitesse de chargement.
- [x] ImageOptim.
- [x] Minification.
- [ ] Sprites/base64 ?
- [x] CSSOptim.
- [ ] Proposition de configuration caching serveur (actuellement hébergé sur GitHub sans cache, quid sur un Apache ou Nginx ?).
- …

**Comment by alvinberthelot** [comment]
Salut,

Je veux bien donner un (petit) coup de main si vous le souhaitez, par contre je vous conseillerai d'avoir un repository source (genre "2015.src") et un autre pour ce qui est buildé (l'existant "2015").

Vous avez un task builder Node.js préféré ou pas ? Brunch, Grunt, Gulp ... ?

A ++

Alvin

**Comment by MattiSG** [comment]
Bonjour Alvin et merci de ta proposition !

Le plus simple me semble être d'utiliser une branche pour la source et de conserver la branche `gh-pages` pour ce qui est généré. Qu'en penses-tu ?

Pas de préférence pour le builder.

**Comment by fredZen** [comment]
Je me permets de suggérer gulp si ça t'est égal, je le trouve moins prise
de tête que ses petits copains.

**Comment by dboissier** [comment]
Salut,

pour les images, je me demandais s'il ne faudrait pas créer 2 tailles (desktop et mobile) afin d’accélérer leur chargement dans un contexte mobile.

**Comment by MattiSG** [comment]
> pour les images, je me demandais s'il ne faudrait pas créer 2 tailles

Absolument, c’est dans #4 mais ça concerne aussi cette issue, tu as raison  :)
